postgres/datadog_checks/postgres/config.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, instance):
         self.pg_stat_activity_view = instance.get('pg_stat_activity_view', 'pg_stat_activity')
         self.statement_samples_config = instance.get('query_samples', instance.get('statement_samples', {})) or {}
         self.statement_metrics_config = instance.get('query_metrics', {}) or {}
+        self.query_settings = instance.get('query_settings', {}) or {}
```

Comment:
```suggestion
        self.query_settings = instance.get('query_settings', {})
```
The or is implicit in the default get, isnt it?